src/main/java/com/booksaw/betterTeams/team/storage/team/SQLTeamStorage.java

```diff
@@ -103,12 +102,11 @@ public List<String> getBanList() {
 		return toReturn;
 	}
 
-	@Override
-	public List<String> getAllyList() {
+	private List<String> getTeamList(final TableName table) {
 
 		List<String> toReturn = new ArrayList<>();
 
-		try (PreparedStatement ps = storageManager.getDatabase().selectWhere("*", TableName.ALLIES,
+		try (PreparedStatement ps = storageManager.getDatabase().selectWhere("*", table,
 				"team1ID LIKE '" + team.getID() + "' OR team2ID LIKE '" + team.getID() + "'")) {
```

Comment:
these column headers only work for the  allies table (i.e. team2ID is only in the allies table), trying to use this statement on different tables will cause issues, why have you moved it to a parameter?

Reply:
As explained above, this is a backport from my own implementation for our server. We have a few more relation types.
So if you want to add them in the future, it'll be very simple.